fuuk/people/translation.py

```diff
@@ -0,0 +1,50 @@
+from modeltranslation.translator import register, TranslationOptions
+
+from fuuk.people.models import Agency, Course, Department, Grant, Human, News, Place, Thesis
+
+
+@register(News)
+class NewsTranslationOptions(TranslationOptions):
+    fields = ('title', 'content')
+    required_languages = {'default': ('title', 'content')}
```

Comment:
Use default language from settings instead of `default`.

This requires adding `MODELTRANSLATION_DEFAULT_LANGUAGE = LANGUAGE_CODE`